crates/re_time_panel/src/lib.rs

```diff
@@ -626,7 +626,7 @@ impl TimePanel {
                         ctx,
                         &InstancePath::from(tree.path.clone()),
                     ))
-                    .exact_width(true),
+                    .truncate(false),
```

Comment:
We don't yet magically detect that we are in the time panel, so we still need to tell the contents not to truncate here